bike-erp/src/components/UserLogs/UserLogs.tsx

```diff
@@ -0,0 +1,61 @@
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Axios from "axios";
+import React, { useEffect, useState } from "react";
+import useStyles from "../Inventory/inventoryStyles";
+
+/*
+    UserLogs page. Admins can view logs and audit trails of all users.
+*/
+const UserLogs: React.FC = () => {
+    const [logTable, setlogTable] = useState<any[]>([]);
+
+  useEffect(() => {
+    Axios.get("http://localhost:3001/userLogs/").then((response) => {
```

Comment:
instead of writing  `http://localhost:3001/userLogs/` you should import the constant in the config file which define the backend URL